src/xbps/files-and-packages.md

```diff
@@ -0,0 +1,35 @@
+# Finding Files and Packages
+
+The `xtools` package contains the
+[xlocate(1)](https://man.voidlinux.org/xlocate.1) utility. `xlocate` works like
+[locate(1)](https://man.voidlinux.org/locate.1), but for files in the Void
+package repositories:
+
+```
+$ xlocate -S
+Fetching objects: 11688, done.
+From https://alpha.de.repo.voidlinux.org/xlocate/xlocate
+ + e122c3634...a2659176f master     -> master  (forced update)
+$ xlocate xlocate
+xtools-0.59_1   /usr/bin/xlocate
+xtools-0.59_1   /usr/share/man/man1/xlocate.1 -> /usr/share/man/man1/xtools.1
+```
+
+It is also possible to use
+[xbps-query(1)](https://man.voidlinux.org/xbps-query.1) to find files, though
```

Comment:
though -> although ?

Reply:
i considered that, and "though" sounds better to me; but if you'd prefer "although", i can certainly change it.

Reply:
I am not entirely sure. If you're okay with "though", it's good for me.